crates/compiler/solve/docs/ambient_lambda_set_specialization.md

```diff
@@ -675,15 +675,15 @@ Type = \region ->
 | Type_function region
 
 Type_function = \region ->
```

Comment:
This one looks like it should not have been removed?

Reply:
I think that's just GitHub syntax highlighting, not a removal diff color. However, I could be missing something.

Reply:
Yes indeed, that is confusing :smile: